pkg/virt-controller/watch/vm.go

```diff
@@ -2605,6 +2650,8 @@ func (c *VMController) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachin
 		}
 	}
 
+	var ifaceHotplugError syncError
```

Comment:
I can't find where `ifaceHotplugError` is being used

Reply:
thanks for cathing that, too many rebases I guess :) I'll remove it.